### PR TITLE
Add Every Door

### PR DESCRIPTION
**Pull request by joelotz**
Adding Every Door mobile editor, also arranged editors in alphabetical order to prevent any bias.

This PR satisfies Issue#91 https://github.com/osmlab/awesome-openstreetmap/issues/91, however, I'm too much of a noob and don't know (yet) how to link the PR to the issue... sorry.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,20 +53,21 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
 
 ### Web Editors
 
+* [Deriviste](https://osm.cycle.travel/deriviste/) - Provides a simple interface to add nodes to OpenStreetMap based on what you see in Mapillary street-level imagery. ([Source Code](https://github.com/systemed/deriviste) / [Wiki](https://wiki.openstreetmap.org/wiki/Deriviste))
+* [Healthsites.io](https://healthsites.io/map) - An online editor focused on adding and improving data on global health facilities. ([Source Code](https://github.com/healthsites/healthsites/) / [Wiki](https://wiki.openstreetmap.org/wiki/Healthsites.io))
```
Comment (Unkn0wnCat) [inline README.md:57]: Please don't do this, we are not soring alphabetically, but by order added.

File: README.md [hunk 2]
```diff
@@ -53,20 +53,21 @@ We also have a list of <a href="UNMAINTAINED.md">unmaintained projects</a>. If y
...
-* [Deriviste](https://osm.cycle.travel/deriviste/) - Provides a simple interface to add nodes to OpenStreetMap based on what you see in Mapillary street-level imagery. ([Source Code](https://github.com/systemed/deriviste) / [Wiki](https://wiki.openstreetmap.org/wiki/Deriviste))
-* [Healthsites.io](https://healthsites.io/map) - An online editor focused on adding and improving data on global health facilities. ([Source Code](https://github.com/healthsites/healthsites/) / [Wiki](https://wiki.openstreetmap.org/wiki/Healthsites.io))
 
 ### Mobile Editors
 
 <!--lint disable no-repeat-punctuation-->
+* [Every Door](https://every-door.app) - Every Door is an Android and iOS app that lets you create and edit objects in OpenStreetMap. It supports editing node and polygon features, and creating point features like shops, benches, and building entrances. ([Source Code](https://github.com/zverik/every_door)/ [Wiki](https://wiki.openstreetmap.org/wiki/Every_Door))
+* [GoMap!!](https://apps.apple.com/app/id592990211) - Feature-rich mobile editor for iOS. (iOS / [Source Code](https://github.com/bryceco/GoMap) / [Wiki](https://wiki.openstreetmap.org/wiki/Go_Map!!))
```
Comment (Unkn0wnCat) [inline README.md:68]: Please move this to the bottom of the block, thanks!

**Review by Unkn0wnCat — CHANGES_REQUESTED** [review]
Thanks for your change, it's already looking great, but it seems you've sorted alphabetically instead of adding to the bottom as is common in awesome-lists. 
Otherwise the PR looks great!

File: README.md (hunk 1 above)
Comment (joelotz) [reply README.md]: Understood. It should be fixed now.

**Review by Unkn0wnCat — APPROVED** [review]
Looking good, everything was updated correctly!

**Comment by Unkn0wnCat** [comment]
Thanks @joelotz for your PR! :tada:

**Comment by joelotz** [comment]
Thanks, I’m learning 😊

I’ll try another one soon.
/joe

**Comment by joelotz** [comment]
Closing https://github.com/osmlab/awesome-openstreetmap/issues/91

**Pull request merged**